src/app/cli/src/services/config_service.rs

```diff
@@ -43,6 +45,8 @@ pub struct CLIConfig {
     /// Data access and visualization configuration
     #[merge(strategy = merge_recursive)]
     pub frontend: Option<FrontendConfig>,
+    /// Users configuration
+    pub users: Option<UsersConfig>,
```

Comment:
merge decorator might be needed in case we ever want to define common set of users for all workspaces within some environment.

Reply:
Missed